.github/workflows/checks.yml

```diff
@@ -132,3 +132,81 @@ jobs:
     - name: Run MCR Client Tests
       run: nix develop --command bash  -c "just mcr-client native build.local.test -t=false"
 
+  move-modules-test:
+    strategy:
+      matrix:
+        include:
+          - os: ubuntu-22.04
+            arch: x86_64
+            runs-on: buildjet-8vcpu-ubuntu-2204
+
+    runs-on: ${{ matrix.runs-on }}
+
+    steps:
+    - name: Checkout repository
+      uses: actions/checkout@v4
+
+    - name: Install Aptos CLI
+      run: |
+        curl -fsSL "https://aptos.dev/scripts/install_cli.py" | python3
+
+    - name: Install Nix
+      uses: DeterminateSystems/nix-installer-action@main
+
+    - name: Run Aptos Tests
+      run: |
+        nix develop --command bash -c "
+          set -e
+          set -x
+          chmod +x .github/scripts/update_move_toml.sh && \
+          ./.github/scripts/update_move_toml.sh && \
+          cd protocol-units/bridge/move-modules && \
+          aptos move test
+        "
+
+  ##solidity-bridge-tests:
```

Comment:
can we remove this commented section?

Reply:
Brought these checks back. We commented them out temporarily.